src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -291,14 +162,14 @@ class VanillaUmiConsensusCaller
     *
     * The same number of base sequences and quality sequences should be given.
     * */
-  private[umi] def consensusCall(reads: Seq[SourceRead]): Option[ConsensusRead] = {
+  private[umi] def consensusCall(reads: Seq[SourceRead]): Option[VanillaConsensusRead] = {
     // check to see if we have enough reads.
     if (reads.size < this.options.minReads) {
```

Comment:
Hrm, I've added the check higher up (mostly so we don't create SourceReads if we don't need to).  But the original author (you I think?) decided there should be a test for this method that passed in insufficient reads and ensured a None is returned.  Leaving it as is for now, though we could convert it to not use Option and eliminate/refactor that test.